sdk/core/core-client-paging-rest/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@azure-rest/core-client-paging",
   "version": "1.0.0-beta.1",
   "description": "A helper library which implements Autorest x-ms-pageable spec for pagination.",
-  "sdk-type": "core",
+  "sdk-type": "client",
```

Comment:
Is this the correct sdk-type? I thought we were planning to create a new "rest" sdk-type for these LLC libraries.

Reply:
@joheredi is that something that is still on your list?